script.js

```diff
@@ -0,0 +1,48 @@
+const titleInput = document.getElementById('title');
+const authorInput = document.getElementById('author');
+const addBtn = document.getElementById('addBtn');
+const bookList = document.getElementById('booksList');
+const books = JSON.parse(localStorage.getItem('books'));
+
+function showBooks() {
+  bookList.innerHTML = '';
+  books.forEach((book, index) => {
+    const bookContainer = document.createElement('div');
+    bookContainer.innerHTML = `
+        <p>${book.title}</p>
+        <p>${book.author}</p>
+        <button class='removeButton' data-index='${index}'>Remove</button>
+        <hr/>
+        `;
+    bookList.appendChild(bookContainer);
+  });
+}
```

Comment:
- [ ] Good job so far, :clap: However, the input data can neither be added to the browser Localstorage nor shown on the browser due to the following error shown below, kindly resolve it. :slightly_smiling_face: 
- [ ] Also remember to use arrays in your Js file that holds the title and author.

![Error](https://github.com/JhonatanErosa/Awesome-Books/assets/43528594/c6620253-fffa-4b20-a8ff-60239e441095)